ItunesSearch/ItunesSearch/SearchResultsTableViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  SearchResultsTableViewController.swift
+//  ItunesSearch
+//
+//  Created by Jonathan Ferrer on 5/14/19.
+//  Copyright © 2019 Jonathan Ferrer. All rights reserved.
+//
+
+import UIKit
+
+class SearchResultsTableViewController: UITableViewController, UISearchBarDelegate {
+
+
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        searchBar.delegate = self
+    }
+
+    func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
+        guard let searchTerm = searchBar.text else { return }
```

Comment:
Would be a good idea to check that the user doesn't send an empty string to the API call 

`guard let searchTerm = searchBar.text, searchTerm != "" else { return }`